test/contracts/workers/set_is_worker.js

```diff
@@ -94,11 +94,11 @@ module.exports.perform = (accounts) => {
   it('pass to set worker if deactivation height is equal to current block number + 1', async () => {
 
     let blockNumber = await web3.eth.getBlockNumber();
-    deactivationHeight = blockNumber + 2; //It is incremented by 2 becoz when u make an smart contract method call using web3's call,it increments by 1 as it simulates what sendTransaction is performing.
+    deactivationHeight = blockNumber + 2; //It is incremented by 2 becoz :- a. block.number returns the pending block number (which is 1 more than the latest block number, which is what is returned by getBlockNumber) and (b) the conditions of the setWorker function that we are testing here.
```

Comment:
My apologies for being unclear here. I did not want you to put my words in—which are opaque, and thus not very helpful (e.g., "the conditions of the setWorker function that we are testing here"), in addition to likely being excessive—but to make a comment that makes it clear to the reader why we are incrementing by 2.

Please take another pass at revising this comment to clarify to the reader concisely and clearly why we are incrementing by 2.